internal/cloud/metadata.go

```diff
@@ -89,6 +90,19 @@ func (i *Identifier) identifyGCP() (bool, error) {
 	return regexp.MatchString(".*Google.*", provider)
 }
 
+func (i *Identifier) identifyNutanix() (bool, error) {
+	log.Debug("Checking if the VM is running on Nutanix...")
+	output, err := i.executor.Exec("dmidecode")
+	if err != nil {
+		return false, err
+	}
+
+	dmidecodeContent := strings.TrimSpace(string(output))
+	log.Debugf("dmidecode output: %s", dmidecodeContent)
+
+	return regexp.MatchString("(?i)nutanix|ahv", dmidecodeContent)
```

Comment:
Maybe we should ask to Bernd what this `ahv` means. 
I know that he gave the result, but `dmidecode` has the option to only send the manufacturer with the `-s system-manufacturer` option. This way, we would only need to check if `Nutanix` is the output, like we do for AWS.
As we don't have the change to test ourselves, I guess that we will need to go with this.

Reply:
`ahv` is the name of Nutanix own hypervisor (but from what i've read, they support multiple hypervisors). Not sure if in all cases it returns `Nutanix` as manufacturer though :/